pkg/test/integration/server.go

```diff
@@ -149,33 +164,54 @@ func New(cfg *rest.Config, o ...Option) (*Manager, error) {
 		op(c)
 	}
 
+	// Create temporary directory for CRDs.
+	dir, err := ioutil.TempDir(".", "")
```

Comment:
It's more typical in Unix environments to create temp dirs under the system temp dir (e.g. `/tmp`). I believe `TempDir` will do this if the dir argument is an empty string. Is there a reason we prefer to create the temp dir under the current directory?

Reply:
I don't believe so. I was actually doing this originally for testing just to see it be created when the test ran.